projects/bp-gallery/src/views/curator/CollectionsPanel.tsx

```diff
@@ -0,0 +1,176 @@
+import { FlatCollection } from '../../types/additionalFlatTypes';
+import { useTranslation } from 'react-i18next';
+import React, { useCallback, useContext, useState } from 'react';
+import { DialogContext, DialogPreset } from '../shared/DialogWrapper';
+import { AlertContext, AlertType } from '../shared/AlertWrapper';
+import {
+  useDeleteCollectionMutation,
+  useGetCollectionInfoByIdQuery,
+  useMergeCollectionsMutation,
+  useUpdateCollectionMutation,
+} from '../../graphql/APIConnector';
+import { useSimplifiedQueryResponseData } from '../../graphql/queryUtils';
+import { Icon, IconButton } from '@mui/material';
+import { Delete, Edit, MergeType } from '@mui/icons-material';
+import AddCollectionMenu from './AddCollectionMenu';
+import UnlinkCollectionAction from './UnlinkCollectionAction';
+
+const CollectionsPanel = ({
+  parentId,
+  onSelectChild,
+}: {
+  parentId: string;
+  onSelectChild: (child: FlatCollection) => void;
+}) => {
+  const { t } = useTranslation();
+  const dialog = useContext(DialogContext);
+  const openAlert = useContext(AlertContext);
+
+  const [selectedChild, setSelectedChild] = useState<string | undefined>(undefined);
+
+  const { data } = useGetCollectionInfoByIdQuery({
+    variables: {
+      collectionId: parentId,
+    },
+    fetchPolicy: 'cache-and-network',
+  });
+  const [updateCollection] = useUpdateCollectionMutation({
+    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
+    errorPolicy: 'all',
+  });
+  const [deleteCollection] = useDeleteCollectionMutation({
+    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
+    errorPolicy: 'all',
+  });
+  const [mergeCollections] = useMergeCollectionsMutation({
+    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
+    errorPolicy: 'all',
+  });
```

Comment:
```suggestion
  const [updateCollection] = useUpdateCollectionMutation({
    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
  });
  const [deleteCollection] = useDeleteCollectionMutation({
    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
  });
  const [mergeCollections] = useMergeCollectionsMutation({
    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
  });
```

Same as above: due to the global setting of the errorPolicy it is now unneccesary here